web-common-marlowe/src/Marlowe/Market/Contract2.purs

```diff
@@ -18,184 +18,138 @@ contractTemplate = { metaData, extendedContract }
-                                    (Role "alice")
-                                    (Party (Role "bob"))
+                                    (Role "Seller")
+                                    (Account (Role "Buyer"))
                                     (Token "" "")
-                                    (ConstantParam "amount")
-                                    Close
-                                )
-                                Close
-                            )
-                            ( When
-                                [ Case
-                                    ( Choice
-                                        ( ChoiceId
-                                            "choice"
-                                            (Role "carol")
-                                        )
-                                        [ Bound one one ]
-                                    )
-                                    Close
-                                , Case
-                                    ( Choice
-                                        ( ChoiceId
-                                            "choice"
-                                            (Role "carol")
-                                        )
-                                        [ Bound one one ]
-                                    )
-                                    ( Pay
-                                        (Role "alice")
-                                        (Party (Role "bob"))
-                                        (Token "" "")
-                                        (ConstantParam "amount")
+                                    (ConstantParam "Price")
+                                    ( When
+                                        [ Case
+                                            ( Choice
+                                                ( ChoiceId
+                                                    "Confirm problem"
+                                                    (Role "Seller")
+                                                )
+                                                [ Bound one one ]
+                                            )
+                                            Close
+                                        , Case
+                                            ( Choice
+                                                ( ChoiceId
+                                                    "Dispute problem"
+                                                    (Role "Seller")
+                                                )
+                                                [ Bound zero zero ]
+                                            )
+                                            ( Pay
+                                                (Role "Seller")
+                                                (Party (PK "0000000000000000000000000000000000000000000000000000000000000000"))
```

Comment:
what is this PK? Shouldn't we use a role?

Reply:
It is to burn money, it is not a participant, it just receives the payment